src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -185,6 +185,22 @@ public function table(array $headers, array $rows)
         $table->setHeaders($headers);
         $table->setRows($rows);
         $table->setStyle($style);
+        $table->setHorizontal($horizontal);
+
+        $table->render();
+        $this->newLine();
+    }
+
+    public function definitionList(array $list)
+    {
+        $style = clone Table::getStyleDefinition('symfony-style-guide');
```

Comment:
Why the `clone` ?

Reply:
do not modify the default style. Anyway I copied the code from the "table"

Reply:
Oh it somehow caches the returned style ? In that case, 👍

Reply:
Just checked and indeed. So 👍